méli-mélo/th-empathy/empathy.html

```diff
@@ -92,3 +92,111 @@ <h4>Code</h4>
 <pre><code>&lt;div class="wb-frmvld provisional wb-steps quiz empathy panel-body"&gt;
     &lt;!-- Steps Quiz Form --&gt;
 &lt;/div&gt;</code></pre>
+
+<h3>Life Journey Home Page Layout</h3>
+<div class="mrgn-tp-lg container">
+	<div class="row wb-eqht-grd">
+		<div class="col-xs-12 col-md-8 empathy main-content small">
+			<div class="row wb-eqht-grd">
+				<div class="col-sm-12 col-md-6 mrgn-bttm-lg">
+					<div class="empathy cards h-100 full-width">
+					<a href="#" class="stretched-link"><h2 class="h4">Preparing for end of life</h6></a>
```

Comment:
Anchors should be inside headings not around them (do for all items). Also, this one has an </h6> closing tag instead of </h2>.

```suggestion
					<h2 class="h4"><a href="#" class="stretched-link">Preparing for end of life</a></h2>
```

Reply:
Comment addressed